jupyter-docker/jupyter_localize_extension.py

```diff
@@ -25,8 +25,13 @@ def _sanitize(self, pathstr):
 
   def _localize_gcs_uri(self, locout, source, dest):
     """Localizes an entry where either the source or destination is a gs: path.
-    Simply invokes gsutil in a subprocess. Quotes paths to remove any shell naughtiness."""
-    cmd = ['gsutil', '-m', '-q', 'cp', '-R', '-c', '-e', pipes.quote(source), pipes.quote(dest)]
```

Comment:
This was basically the issue -- `pipes.quote` should not be used in `subprocess.call` where `Shell=False`.